t/40-script_load_templates.t

```diff
@@ -55,15 +67,31 @@ $args      = "--host $host --apikey $apikey --apisecret $apisecret $filename";
 my $expected = qr/JobGroups.+=> \{ added => 1, of => 1 \}/;
 test_once $args, $expected, 'Admin may load templates', 0, 'successfully loaded templates';
 test_once $args, qr/group with existing name/, 'Duplicate job group', 255, 'failed on duplicate job group';
-kill TERM => $pid;
 
 $mojoport = Mojo::IOLoop::Server->generate_port;
 $host     = "localhost:$mojoport";
-$pid      = OpenQA::Test::Utils::create_webapi($mojoport, sub { OpenQA::Test::Database->new->create; });
+$schema   = OpenQA::Test::Database->new->create();
```

Comment:
Do you know that you could pass the option `skip_fixtures => 1`? This would help to bring down schema initialization runtime a lot.

Reply:
Yes, [I was doing that before](https://github.com/os-autoinst/openQA/compare/13b401a8eb441f345ca1d810eb3f564d99e1f37c..11bbcbac49d5432c3f5e4d3bdd45ce80e4a51e5d).

I'm dropping the test for a broken JSON file. Without it there's no side effects to the other tests and no need for an extra server.